<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ronak" failures="2" tests="4" name="Test" time="63.745" errors="0" timestamp="2025-08-02T20:58:26 IST">
  <testcase classname="testCases.TC003_LoginDatadriven_Factory" name="dataDrivenTest" time="11.416">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDatadriven_Factory.dataDrivenTest(TC003_LoginDatadriven_Factory.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- dataDrivenTest -->
  <testcase classname="testCases.TC003_LoginDatadriven_Factory" name="dataDrivenTest" time="2.371"/>
  <testcase classname="testCases.TC003_LoginDatadriven_Factory" name="dataDrivenTest" time="2.226"/>
  <testcase classname="testCases.TC003_LoginDatadriven_Factory" name="dataDrivenTest" time="11.266">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDatadriven_Factory.dataDrivenTest(TC003_LoginDatadriven_Factory.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- dataDrivenTest -->
</testsuite> <!-- Test -->
